### PR TITLE
Updated Kudos icon in nav

### LINKED ISSUE #2961: Nav: Change icon of 'Send Kudos' and 'My Grants' on nav dropdown
### Current Behavior

Icons of 'Send Kudos' and 'My Grants' on nav dropdown do not represent the branding of the feature
screenshot 2018-11-28 01 17 30

### Expected Behavior
Change the icons of 'Send Kudos' and 'My Grants' following the screenshot below:
screenshot 2018-11-28 01 16 26

Sketch files (new icons inside):
https://drive.google.com/open?id=1JLMtaFe3ul7nyWgH4q_1cxCIFOImUUEV

### PR DESCRIPTION
**Pull request by usmanmuhd**
Fixes #2961 (only Kudos icon)
image

### CODE REVIEW

**Comment by usmanmuhd** [comment]
@thelostone-mc Could you take a look now?

File: app/dashboard/templates/shared/nav_auth.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
           {% trans "Send Tip" %}
         </a>
         <a class="dropdown-item" href="{% url "kudos_marketplace" %}">
-          <i class="fas fa-magic"></i>
+          <i class="fas"><img src="{% static "v2/images/kudos/IconKudos.svg" %}"></i>
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/nav_auth.html]: @usmanmuhd could we remove the fas class too ?

Comment (usmanmuhd) [reply app/dashboard/templates/shared/nav_auth.html]: Here as well it is required to keep the icon aligned in the same way as the other icons.

Comment (thelostone-mc) [reply app/dashboard/templates/shared/nav_auth.html]: Ah ok 👍

**Pull request merged**